broker/src/main/java/org/apache/rocketmq/broker/transaction/TransactionStore.java

```diff
@@ -18,7 +18,7 @@
 package org.apache.rocketmq.broker.transaction;
 
 import java.util.List;
-
+@Deprecated
```

Comment:
？

Reply:
This interface has been deleted in the latest commit.